docker/sonarqube/README.md

```diff
@@ -0,0 +1,40 @@
+## SonarQube
+
+## Preliminary Steps
+(you *will* need to run this daily before attempting to start sonarqube - use precompose.sh script in case other things are needed)
+sudo sysctl -w vm.max_map_count=262144
+sudo sysctl -w fs.file-max=131072
+
+sudo vi /etc/sysctl.conf
+vm.max_map_count=262144
+fs.file-max=131072
+
+## Start SonarQube
+In docker/sonarqube directory, run:
+```bash
+./precompose.sh
+docker-compose up -d
+```
+
+## Setup SonarQube
+go to localhost:9000 to view the sonarqube app
+admin/admin is the default password (change it when you first access it)
+
+Set project name to DataWave
+scan locally
+generate a token (never expires) -- SAVE IT
+Create a sonar.sh script, add it to your ~/bin dir (or anywhere that is on your configured path)
+```bash
+mvn clean verify
+mvn -e sonar:sonar \
+  -Dsonar.projectKey=Datawave \
+  -Dsonar.host.url=http://localhost:9000 \
+  -Dsonar.login=<TOKEN>
+```
+
```

Comment:
Can you add to the readme the one line:

```
mvn clean install && mvn -e sonar:sonar -Dsonar.projectKey=Datawave -Dsonar.host.url=http://localhost:9000 -Dsonar.login=<TOKEN> -Dsonar.exclusions=**/*.java
```

So we can copy and paste it easily.

Reply:
added exclusion & reduced to one line